src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
The method `combineSlices` is not a standard method provided by Redux Toolkit. Make sure that this method is correctly implemented or imported. Typically, `combineReducers` is used to combine multiple slices in Redux Toolkit.